frontend/src/v5/ui/controls/dueDate/dueDateWithIcon/dueDateWithIcon.component.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { formatDate } from '@/v5/services/intl';
+import { FormInputProps } from '@controls/inputs/inputController.component';
+import { Tooltip } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { BaseDueDate } from '../baseDueDate.component';
+import { CalendarIcon, DueDateWithIconContainer } from '../dueDate.styles';
+import { DateContainer, EmptyDateContainer } from '../dueDateWithLabel/dueDateLabel/dueDateLabel.styles';
+
+type IDueDateWithIcon = FormInputProps & {
+	tooltip?: string;
+	onBlur: () => void;
+};
+
+export const DueDateWithIcon = ({ value, disabled, tooltip, ...props }: IDueDateWithIcon) => {
+	const isOverdue = value < Date.now();
+	const formattedDate = formatDate(value, {
+		day: 'numeric',
+		month: 'numeric',
+		year: '2-digit',
```

Comment:
I think we could change this to a constant to `shortDate` or a new function in intl for shortDate